### PR TITLE
French localization of nojob_fr.properties

### PR DESCRIPTION
**Pull request by SoleneGK**
French localization of home page when there are no jobs
Sans titre

### CODE REVIEW

**Comment by mmh891113** [comment]
@SoleneGK you can create a sigle plugin for French language, 
See JEP-216 https://github.com/jenkinsci/jep/blob/master/jep/216/README.adoc
there are two plugin for reference:
https://github.com/jenkinsci/localization-zh-cn-plugin
https://github.com/jenkinsci/localization-tr-plugin

**Comment by aHenryJard** [comment]
@mmh891113 thank you I was not aware about this JEP. I think that this PR can be merged and the property file moved later when the plugin will exists (I am the one that proposed french translation to @SoleneGK in order to discover Jenkins during hacktober fest).

File: core/src/main/resources/hudson/model/AllView/noJob_fr.properties
Comment (PierreBtz) [inline core/src/main/resources/hudson/model/AllView/noJob_fr.properties:27]: On pourrait peut être aligner tous les "setup" et réutiliser "configurer" comme pour `setUpCloud`?

```suggestion
setUpDistributedBuilds=Configurer un build distribu\u00e9
```

File: core/src/main/resources/hudson/model/AllView/noJob_fr.properties
Comment (PierreBtz) [inline core/src/main/resources/hudson/model/AllView/noJob_fr.properties:28]:
```suggestion
setUpAgent=Configurer un agent
```

**Review by PierreBtz — APPROVED** [review]
lgtm once we have aligned all the 'setups' to use the same phrasing.

**Review by oleg-nenashev — APPROVED** [review]
The changes look good to me, thanks!
Creating a French localization plugin might be a good option if there is an active localization community.

We may merge it in 24 hours if there is no negative feedback. Please see the [merge process documentation](https://github.com/jenkinsci/jenkins/blob/master/docs/MAINTAINERS.adoc#merge-process) for more information about the merge process

**Pull request merged**
